api/python/test/canary/teams.py

```
@@ -1,6 +1,8 @@
 from ai.chronon.api.ttypes import Team
 from ai.chronon.repo.constants import RunMode
-from ai.chronon.types import ConfigProperties, EnvironmentVariables
+from ai.chronon.types import ConfigProperties, EnvironmentVariables, ClusterConfigProperties
+from ai.chronon.repo.cluster import generate_dataproc_cluster_config
+from build.lib.ai.chronon.repo.constants import RunMode
```

Ruff (0.11.9)

Redefinition of unused `RunMode` from line 2

(F811)